independent-projects/tools/devtools-common/src/main/java/io/quarkus/devtools/commands/handlers/CreateProjectCommandHandler.java

```diff
@@ -115,11 +160,12 @@ public QuarkusCommandOutcome execute(QuarkusCommandInvocation invocation) throws
                     .example(invocation.getValue(EXAMPLE))
                     .noCode(invocation.getValue(NO_CODE, false))
                     .addCodestarts(invocation.getValue(EXTRA_CODESTARTS, Collections.emptySet()))
-                    .noBuildToolWrapper(invocation.getValue(NO_BUILDTOOL_WRAPPER, false))
+                    .noBuildToolWrapper(parent == null && invocation.getValue(NO_BUILDTOOL_WRAPPER, false))
```

Comment:
@ia3andy wdyt .noBuildToolWrapper(invocation.getValue(NO_BUILDTOOL_WRAPPER, parent != null))

I dont know if it makes sense

Reply:
yeah I think `.noBuildToolWrapper(invocation.getValue(NO_BUILDTOOL_WRAPPER, parent != null))` is better as it still let the user choose.